khmer/_khmer.cc

```diff
@@ -4575,16 +4575,22 @@ static PyObject * hllcounter_consume_fasta(khmer_KHLLCounter_Object * me,
         PyObject * args)
 {
     const char * filename;
+    PyObject * output_records_o = NULL;
+    bool output_records = false;
 
-    if (!PyArg_ParseTuple(args, "s", &filename)) {
+    if (!PyArg_ParseTuple(args, "s|O", &filename, &output_records_o)) {
```

Comment:
Should `output_records` be a keyword on this method? I think it makes more sense, because calling it is a bit weird (check https://github.com/dib-lab/khmer/pull/1204/files#diff-7f09af630bd7b293c0827bba1d411617R120 for an example).

This also raises the question: do we want to do this in other places? We do have a lot of default arguments in the extension glue code that are not exposed nicely to Python land...